pgmpy/models/BayesianModel.py

```diff
@@ -410,9 +412,9 @@ def active_trail_nodes(self, start, observed=None):
         >>> student.add_nodes_from(['diff', 'intel', 'grades'])
         >>> student.add_edges_from([('diff', 'grades'), ('intel', 'grades')])
         >>> student.active_trail_nodes('diff')
```

Comment:
@raghavg7796 Earlier we used to support only strings for node names but later we refactored the code to accept any hashable object. So, we might have missed changing the docstring somewhere. Can you create an issue for that ?